odd-collector/odd_collector/adapters/mssql/mappers/tables.py

```diff
@@ -18,7 +18,7 @@ def map_table(table: Table, generator: MssqlGenerator):
     oddrn = generator.get_oddrn_by_path("tables")
 
     map_col = partial(map_column, oddrn_generator=generator, parent_oddrn_path="tables")
-    dataset = DataSet(field_list=lmap(map_col, table.columns))
+    dataset = DataSet(rows_number=table.table_rows,field_list=lmap(map_col, table.columns))
```

Comment:
Hello, @bbsankarsh. Overall LGTM.
But your code needs to be linted. To do so, you should invoke command `make lint` in the root `odd-collectors/` directory. There are some missed spaces and blank lines that are required by PEP8 standard.